assets/js/modules/analytics/components/settings/PropertyOrWebDataStreamNotAvailableError.js

```diff
@@ -71,9 +71,7 @@ export default function PropertyOrWebDataStreamNotAvailableError( props ) {
 	);
 
 	const getPropertiesError = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS_4 ).getErrorForSelector( 'getProperties', [
-			accountID,
-		] )
+		select( MODULES_ANALYTICS_4 ).getErrorForSelector( 'getAccounts', [] )
```

Comment:
Hm... the problem here is that now we use the account summaries endpoint to get properties and if that endpoint return an error, that error is not always going to be related to a properties issue. So, there is a chance that we will display a false positive error. I am not sure if there is a way to mitigate this or not... What do you think? cc @aaemnnosttv / @techanvil  / @tofumatt

Reply:
As mentioned [above](https://github.com/google/site-kit-wp/pull/7966/files#r1433028182), the `getAccounts` error was previously being set manually (although the GA4 version is not being set in this iteration).

We could take a similar approach here though, and manually set an error in the `getProperties()` resolver if the account summaries endpoint returns an error. The we'll have a correctly scoped error, and can revert this change.

Reply:
Won't it have the same effects even if we create a `getPropertySummaries` resolver and manually set an error as it will call the `getAccountSummaries` selector?

Reply:
@hussain-t, the point of setting an explicit `getProperties` or `getPropertySummaries` error would be so we could have an error explicitly scoped to the property-related selector - if we're instead checking for the `getAccountSummaries` error, we could _potentially_ run into a scenario where, say, `getPropertySummaries()` hasn't actually been called, but `getAccountSummaries()` has and results in an error, and so we show an error when we shouldn't.

Admittedly, it does seem fairly unlikely to occur in practice but seeing as @eugene-manuilov has raised this and we do have a potential mitigation it's worth considering. It would be good to get Eugene's thoughts on this too.

Reply:
My point at the beginning was a bit different but now after re-reviewing it I think that Hussain updated that component correctly and we can resolve this conversation. We still can check for the `getAccountSummaries` error because it is called internally in the `getPropertySummaries` selector. So, I think there is no need to set that error for the `getPropertySummaries` selector manually.

Thanks @techanvil and @hussain-t.